source/07-processing/07-federated-searches.rst

```diff
@@ -17,6 +17,7 @@ The most common ways to recognize federated and automated search activity are as
 * If the standard HTML interface is being used (e.g. for screen scraping), the browser ID within the web log files can be used to identify the activity as coming from a federated search.
 * All searches via APIs and `Z39.50 activity <http://www.niso.org/standards/resources/Z39.50_Resources>`_ must be counted as Searches_Federated, as the results are not presented in the platform user interface.
 * If an API gateway is available, set up an instance of the gateway that is for the exclusive use of federated search tools. It is recommended you also require the federated search engine to include an identifying parameter when making requests to the gateway.
+* For Z39.50 activity, authentication is usually through a username/password combination. Create a unique username/password that just the federated search engine will use.
```

Comment:
I don't mind keeping these points, but since all searches via APIs and Z39.50 are Searches_Federated anyway, there is no need to identify specific clients? For Z39.50 some content providers offer IP authorization, and that's of course fine.